pepper-apis/docs/specification/src/endpoints/user.studies.activity.uploads.yml

```diff
@@ -44,13 +44,15 @@ post:
               description: The question identifier
             fileName:
               type: string
-              maxLength: 100
+              maxLength: 255
               description: Name of file to be uploaded
             fileSize:
               type: integer
+              minimum: 1
               description: Size of the file, in bytes (subject to a limit such as 2MB)
             mimeType:
               type: string
+              maxLength: 255
```

Comment:
Note that I made some changes to the API docs to reflect updated implementation.